src/commands/leaderboard.ts

```diff
@@ -0,0 +1,27 @@
+import { Command } from 'discord-akairo';
+import { Message } from 'discord.js';
+import Repository from '../repository/Repository';
+import { getEmbedFromLeaderboardList } from '../util';
+
+export default class LeaderboardCommand extends Command {
+    constructor() {
+        super('leaderboard', {
+           aliases: ['leaderboard']
+        });
+    }
+
+    async exec(message: Message) {
+        const { author } = message;
+        const list = await Repository.getLeaderboardInfo();
+        const userKarma: Record<string, number> = {};
```

Comment:
Use `reduce` instead of `forEach` to populate this.

```ts
const userKarma = list.reduce(
  (acc, item) => ({
    ...acc,
    [item.user.id]: (acc[item.user.id] || 0) + item.karma,
  }),
  {} as Record<string, number>
);
```